src/ol/geom/circle.js

```diff
@@ -186,7 +186,7 @@ ol.geom.Circle.prototype.setCenter = function(center) {
  */
 ol.geom.Circle.prototype.setCenterAndRadius = function(center, radius, opt_layout) {
   if (!center) {
-    this.setFlatCoordinates(ol.geom.GeometryLayout.XY, null);
+    this.setFlatCoordinates(opt_layout || ol.geom.GeometryLayout.XY, null);
```

Comment:
But if there are no coordinates, layout should not matter right? I took them all out and all tests passed. setLayout is called when a true coordinates argument is provided or am I missing something?